riff-cli/cmd/delete.go

```diff
@@ -123,20 +129,20 @@ func deleteResources(cmdArgs []string, actingClient kubectl.KubeCtl) error {
 	return err
 }
 
-func lookupTopicNames(opts DeleteOptions, queryClient kubectl.KubeCtl) (string, string, error) {
+func lookupByTopicBindingName(opts DeleteOptions, queryClient kubectl.KubeCtl) (string, string, string, error) {
 	getArgs := []string{"get"}
 	if opts.Namespace != "" {
 		getArgs = append(getArgs, "--namespace", opts.Namespace)
 	}
-	getArgs = append(getArgs, "functions.projectriff.io", opts.FunctionName, "-o", "json")
+	getArgs = append(getArgs, "topicbindings.projectriff.io", opts.FunctionName, "-o", "json")
 	json, err := queryClient.Exec(getArgs)
 	if err != nil {
-		return "", "", err
+		return "", "", "", err
 	}
 	parser := jsonpath.NewParser([]byte(json))
 
+	handler, _ := parser.StringValue(`$.spec.handler`)
```

Comment:
this should be `$.spec.function` or the function resources will not get deleted, should also rename the variable I guess